Intersect.Server/Entities/Player.cs

```diff
@@ -1747,7 +1806,27 @@ public void UseItem(int slot, Entity target = null)
             }
         }
 
-        public bool TakeItemsBySlot(int slot, int amount)
+        /// <summary>
+        /// Add or Remove items through a simple switch with this method.
+        /// </summary>
+        /// <param name="itemId"></param>
+        /// <param name="amount"></param>
+        /// <param name="add"></param>
+        /// <param name="sendUpdate"></param>
+        /// <returns></returns>
+        public bool ChangeItemById(Guid itemId, int amount, bool add, bool sendUpdate = true)
```

Comment:
Since it only references Try methods it should also be named Try.

Reply:
Ah, must have missed that one. Changed it!